sympy/core/tests/test_args.py

```diff
@@ -35,7 +35,7 @@ def test_all_classes_are_tested():
             if not file.endswith(".py"):
                 continue
 
-            with io.open(os.path.join(root, file), "r", encoding='utf-8') as f:
+            with open(os.path.join(root, file), "r", encoding='utf-8') as f:
```

Comment:
You may have to resolve the flake8 import issues here

Reply:
Done, thanks